c10/core/impl/LocalTensorTypeSet.cpp

```diff
@@ -14,29 +14,55 @@ namespace {
 
 // NB: Zero initialized!
 thread_local uint64_t raw_excluded;
+thread_local uint64_t raw_included;
 
 #else // defined(CAFFE2_FB_LIMITED_MOBILE_CAPABILITY)
 
 uint64_t raw_excluded = 0;
+uint64_t raw_included = 0;
 
 #endif
 
+void set_enabled_via_excluded(TensorTypeId tid, bool enabled) {
```

Comment:
The names of these functions are confusing to me. The caller still needs to understand that thefinal set will be `(tensor_properties | included) - excluded` but then there is also a second translation between how this function affects included/excluded. Wouldn't it make more sense to just have it follow the set operations directely, excluded.add, excluded.remove, included.add, included.remove.